telcell/celldb/cell_collection.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+from abc import abstractmethod
+import datetime
+from typing import Optional, Iterable, Sized
+
+import geopy
+
+from telcell.cell_identity import CellIdentity
+
+
+class Properties(dict):
```

Comment:
wat is dit en waarom kan je de cellcollection niet gewoon een Iterable[dict] laten zijn?

Reply:
Dit zorgt ervoor dat je `cell.azimuth` kan typen in plaats van `cell["azimuth"]`